src/styles/_notifications.less

```diff
@@ -8,7 +8,11 @@
 }
 
 .toast-notifications-list-pf {
+  @media(max-width: @screen-xxs-max) {
+    max-width: calc(100% - - 40px); // Set max-width when < 769, so that word-break functions on Firefox & IE.
```

Comment:
the 40px accounts for the 20px left & right margin from `.container-fluid` 

Found that these are equal, but I'll switch to the other
calc(100% - - 40px) === calc(~"100% - 40px")

meant to use var @screen-xs-max, will update.